sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -20,6 +20,26 @@ and includes six main functions:
 - [Azure Subscription][azure_subscription]
 - [Cognitive Services or Text Analytics account][text_analytics_account] to use this package.
 
+### Include the Package
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-textanalytics</artifactId>
+    <version>5.1.0</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+**Note:** This version of the client library defaults to the `v3.1` version of the service.
+
+This table shows the relationship between SDK services and supported API versions of the service:
+
+|SDK version|Supported API version of service
+|-|-
+|5.0.x | 3.0
```

Comment:
Do we need to mention the service version here?

Reply:
I mimic the same pattern from Form Recognizer: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/README.md#include-the-package

Reply:
Misunderstood the table, you're good.